routes/basic_db_functions_route.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from DAL.basic_db_functions import BasicDbFunctions
+
+router = APIRouter()
+
```

Comment:
remove 1 line space

Reply:
I use PyCharm format option for formatting files and it doesn’t give me any warning when loading those files.